server/services/meross/lib/plug/plug.getState.js

```diff
@@ -0,0 +1,20 @@
+const { STATE } = require('../../../../utils/constants');
+const { getDeviceParam } = require('../../../../utils/device');
+const { merossStatePayload } = require('../../utils');
+
+/**
+ * @private
+ * @description Get the current state of a device.
+ * @param {Object} device - The device object.
+ * @param {Object} deviceFeature - The deviceFeature object.
+ * @returns {Promise} Resolving with deviceFeature state.
+ * @example
+ * getState(deviceFeature);
+ */
+async function getState(device, deviceFeature) {
+  const url = getDeviceParam(device, 'CAMERA_URL');
```

Comment:
Why CAMERA_URL ?

Reply:
copy/past from rtsp-camera i guess.. 
I'll fix that